lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -3721,6 +3722,30 @@ instance IsServerError ErrBalanceTx where
                 ]
         ErrBalanceTxSelectAssets err -> toServerError err
         ErrBalanceTxAssignRedeemers err -> toServerError err
+        ErrBalanceTxNotYetSupported ConflictingNetworks ->
+            apiError err403 CreatedInvalidTransaction $ mconcat
+                [ "There are withdrawals for multiple networks (e.g. both "
+                , "mainnet and testnet) in the provided transaction. This "
+                , "makes no sense, and I'm confused."
```

Comment:
```suggestion
                , "mainnet and testnet) in the provided transaction. This "
                , "is not supported."
```

Reply:
I thought I'd be consistent with the other anthropomorphised error-messages 😂